src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ name, age, isMarried, partnerName, sex }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+    {age && (
+      <p className="Person__age">{`I am ${age}`}</p>
+    )}
+
+    <p className="Person__partner">
+      {isMarried === true
+        ? `${partnerName} is my ${sex === 'f' ? 'husband' : 'wife'}`
+        : 'I am not married'}
+    </p>
```

Comment:
I think that each part should be inside the tag `<p className="Person__partner"> <p>`

Reply:
u mean 3 <p className="Person__partner"> <p> instead 1?

Reply:
not agree